### PR TITLE
remove sqlsmith patch

### PR DESCRIPTION
**Pull request by Tmonster**
Removing sqlsmith patch as it has been applied in 

https://github.com/duckdb/duckdb_sqlsmith/pull/19

### CODE REVIEW

**Review by carlopi — CHANGES_REQUESTED** [review]
I think you need to remove the APPLY_PATCHES line from `.github/config/out-of-tree.cmake` and same from the custom cmake in duckdblabs/duckdb-fuzzer-ci.

File: .github/config/out_of_tree_extensions.cmake [hunk 1]
```diff
@@ -118,7 +118,6 @@ duckdb_extension_load(sqlite_scanner
 duckdb_extension_load(sqlsmith
         GIT_URL https://github.com/duckdb/duckdb_sqlsmith
         GIT_TAG 721460ff1f31ce1dc1e4a9c4a55c0faf0b466dcb
```
Comment (carlopi) [inline .github/config/out_of_tree_extensions.cmake:120]:
```suggestion
        GIT_TAG 9729cade201987b22ec7fc4b700f6d34e1e4111f
```
Forgot, also this needs to be bumped to AFTER absorbing the patches.

**Comment by carlopi** [comment]
> Done. @carlopi , if there are no patches, does apply patches then fail the build?

Yes, if `APPLY_PATCHES` is present some patches need to available.

**Comment by Tmonster** [comment]
I imagine the extension builds are failing because they attempt to build sqlsmith with patches

**Comment by Mytherin** [comment]
Thanks!

**Pull request merged**